pages/jobs.md

```diff
@@ -1,10 +0,0 @@
----
-layout: page-fullwidth
-title: "Job Opportunities related to HPC Carpentry"
```

Comment:
It's not impossible...

Reply:
Restored by 62b97559, new content in 279e9729.